tap/extensions/http/Makefile

```diff
@@ -0,0 +1,16 @@
+skipbin := $$(ls -A --ignore='.??*' bin)
```

Comment:
I understand that those options of --ignore are not exists in mac 
(https://github.com/up9inc/mizu/pull/749#discussion_r801828585) 
can we change those to something that will be supported in both mac and linux so developers will be able to use it on their laptop? 
(maybe "find")?

Reply:
@gadotroee done with https://github.com/up9inc/mizu/commit/380390d6d05fdfdf220bb4b4e7c3bf919cda1049